lib/src/select/Select.test.js

```diff
@@ -293,26 +278,45 @@ describe("Select component tests", () => {
     expect(getByText("Option 02, Option 03, Option 04, Option 06")).toBeTruthy();
     expect(submitInput.value).toBe("4,2,6,3");
   });
-  test("Disabled select - Clear all options action must be shown but not clickable", () => {
-    const { getByRole, getByText, queryByRole } = render(
-      <DxcSelect label="test-select-label" value={["1", "2"]} options={single_options} disabled searchable multiple />
+  test("Disabled select - Listbox cannot be opened by click or keyboard press", () => {
```

Comment:
Change the name of the test since there is no longer any keboard press.

Reply:
Done!